uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/request-modal/details/leave.html

```diff
@@ -5,13 +5,13 @@
       <div class="col-xs-8">
         <div class="btn-group btn-group-justified">
           <label
-            ng-disabled="$ctrl.isMode('view') || ($ctrl.isRole('manager') && !$ctrl.isLeaveType('sickness'))"
+            ng-disabled="$ctrl.isMode('view') || (!$ctrl.isMode('create') && !$ctrl.isLeaveType('sickness'))"
```

Comment:
before it was:
"disable if we're in view mode, or if the user is a manager and we are not looking at a sickness leave request"

now it's:
"disable if we're in view mode, or if the mode is not create and we are not looking at a sickness leave request"

two questions:
1. Why the change from "is a manager" to "it's not create mode"?
2. Why the focus on the sickness type?